ihatemoney/web.py

```diff
@@ -327,16 +327,18 @@ def remind_password():
             # get the project
             project = Project.query.get(form.id.data)
             # send a link to reset the password
-            current_app.mail.send(
-                Message(
-                    "password recovery",
-                    body=render_localized_template(
-                        "password_reminder", project=project
-                    ),
-                    recipients=[project.contact_email],
-                )
+            remind_message = Message(
+                "password recovery",
+                body=render_localized_template("password_reminder", project=project),
+                recipients=[project.contact_email],
             )
-            return redirect(url_for(".password_reminder_sent"))
+            result = send_email(
+                remind_message,
+                _("A password reminder email has just been sent to you"),
+                _("Error while sending password reminder email"),
```

Comment:
I wonder about the message here. Should we be more user friendly, with a message like "Sorry, an error occured when we tried to send you a reminder email, try again and if the problem persist, try contacting the instance administrators", or something like it.

It's longer, for sure, but I feel it makes things more human :-)

Reply:
User-facing messages are always the hardest part :) I will try to improve it.